res/xml/lawnchair_desktop_preferences.xml

```diff
@@ -84,6 +84,18 @@
 
     </ch.deletescape.lawnchair.preferences.StyledPreferenceCategory>
 
+    <ch.deletescape.lawnchair.preferences.StyledPreferenceCategory
+        android:key="prefCat_desktopFolders"
+        android:title="@string/pref_appcategorization_folders_title">
```

Comment:
I wasn't sure about this part, I took the "Folders" label from an unrelated section of the localization strings.